client/README.md

```diff
@@ -0,0 +1,5 @@
+# Client go module
```

Comment:
A top level README.md with instructions to users on how to import this go module would be great too! Can be done as a separate PR once there's actually something to import.